requirements/app/test.txt

```diff
@@ -12,3 +12,4 @@ isort>=5.0
 mypy>=0.720
 httpx
 trio
+pympler
```

Comment:
These bots trying to add new deps. Sneaky!

Reply:
I wonder why this happened. Looks like we would not need `pympler ` right?.

Reply:
it was a merge artefact :)